config.md

```diff
@@ -2,6 +2,7 @@
 
 The container's top-level directory MUST contain a configuration file called `config.json`.
 The canonical schema is defined in this document, but there is a JSON Schema in [`schema/config-schema.json`](schema/config-schema.json) and Go bindings in [`specs-go/config.go`](specs-go/config.go).
+The media type for this file is configuration is `application/vnd.oci.runtime.config.v1+json`, and all specifications using that media type will require [JSON](glossary.md#json) objects with an [`ociVersion`](#specification-version).
```

Comment:
I'm suggesting the text `for this file is configuration is` includes a typo.  And was hoping for a link to something that describes the media types.  Should've looked it up, pretty sure I saw it somewhere around here :)